threedi_tools.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, iface):
         self.about_tool = About(iface)
         self.cache_clearer = CacheClearer(iface, self.ts_datasource)
         self.result_selection_tool = ThreeDiResultSelection(iface, self.ts_datasource)
-        self.toolbox_tool = CommandBox(iface, self.ts_datasource)
+        self.toolbox_tool = CommandBox(iface, self.ts_datasource, self.plugin_dir)
```

Comment:
Wordt die plugin_dir überhaupt wel echt gebruikt in die CommandBox? Ik heb 'em op veel plekken in m'n andere PR al weggehaald.

Liever niet op deze manier doorgeven als 'ie toch wel nodig is. Kan je daar `from ThreeDiToolbox.dependencies import OUR_DIR` gebruiken? In een ander PR kunnen we er dan wel `PLUGIN_DIR` van maken.